.github/workflows/release.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       - run: echo "$HOME/.deno/bin" > $GITHUB_PATH
 
       - name: Build
-        run: deno --unstable run -A ./cmd/build.ts $VERSION
```

Comment:
why?

Reply:
It's deprecated
https://docs.deno.com/runtime/reference/cli/unstable_flags/#--unstable